src/plugins/python/reporter_bugzilla.py

```diff
@@ -848,7 +848,7 @@ def log_out(bug_info, rhbz, dump_dir_name):
 
         dup_comment = False
         trimmed_comment = re.sub(r'[ \t\n]', r'', bzcomment)
-        for c in bug_info['bi_comments']:
+        for c in bug_info.get('bi_comments') or []:
```

Comment:
Why not just `bug_info.get('bi_comments', [])`?

Reply:
Because the key `bi_comments` can exist, but the value can be `None`. The default value would be used only if the key doesn't exist.

Reply:
Ah, OK. Could you perhaps add a short comment noting that `bi_comments` can be `None` and hence the `or`?

Reply:
Good idea ;) I will probably get to that tomorrow though. No rush here :)

Reply:
Updated (finally) ;)